Gem/Code/Source/Components/NetworkMatchComponent.h

```diff
@@ -73,6 +82,12 @@ namespace MultiplayerSample
             RoundTickOnceASecond();
         }, AZ::Name("NetworkMatchComponentController")};
 
+        void RestTickOnceASecond();
```

Comment:
Is this server only section?

Reply:
Fixed! Moved both RestTick and RoundTick to #if SERVER